disease/cli.py

```diff
@@ -1,14 +1,27 @@
 """This module provides a CLI util to make updates to normalizer database."""
+from timeit import default_timer as timer
+from os import environ
+
 import click
-from disease import SOURCES_CLASS_LOOKUP, SOURCES_LOWER_LOOKUP, logger
-from disease.schemas import SourceName
-from disease.database import Database, confirm_aws_db_use
-from disease.etl.mondo import Mondo
-from disease.etl.merge import Merge
 from botocore.exceptions import ClientError
 from boto3.dynamodb.conditions import Key
-from timeit import default_timer as timer
-from os import environ
+
+from disease import SOURCES_LOWER_LOOKUP, logger
+from disease.schemas import SourceName
+from disease.database import Database, confirm_aws_db_use, SKIP_AWS_DB_ENV_NAME, \
+    VALID_AWS_ENV_NAMES, AWS_ENV_VAR_NAME
+from disease.etl.merge import Merge
+from disease.etl.mondo import Mondo
+from disease.etl import NCIt  # noqa: F401
+from disease.etl import DO  # noqa: F401
+from disease.etl import OncoTree  # noqa: F401
+from disease.etl import OMIM  # noqa: F401
```

Comment:
Can this be a single import statement (and is something goofed about Mondo that won't let it import from the toplevel `disease.etl` namespace)?

Reply:
Oh ya, sorry I just copied/pasted from what was in disease/__init__.py

Reply:
Yeah, that's what I figured. I can tweak it in a future PR, no biggie

Reply:
Wow, not all heroes wear capes! Thanks.